src/HotChocolate/Core/src/Types.Scalars/WellKnownScalarTypes.cs

```diff
@@ -5,5 +5,7 @@ internal static class WellKnownScalarTypes
         public const string NonEmptyString = nameof(NonEmptyString);
         public const string PositiveIntType = nameof(PositiveIntType);
         public const string NegativeInt = nameof(NegativeInt);
+
+        public const string EmaillAddress = nameof(EmaillAddress);
```

Comment:
Should this be `EmailAddress` or `EmailAddressType`?

Reply:
`EmailAddress` but with one `l` :D

Reply:
EmailAddressType

@PascalSenn we should be consistent and add the Type postfix everywhere.

Reply:
Ah I messed the PositiveIntType up. 
Should we add `Type`? 
I mean, StringType is also just Type

Reply:
Consistency is key. We have named all the scalars with this suffix that we should not just introduce a new pattern. Every type ends with Type ... NonNullType, StringType, ListType .... and so on.

Reply:
After talking to pascal, this is the name of the scalar ... So we need to do it without Type postfix.

Reply:
@PascalSenn The reason I asked is I was following the `NonEmptyString` and here https://github.com/wonbyte/hotchocolate/blob/d2592fee00776363b26bd8361590e5b6e5fc5cb9/src/HotChocolate/Core/src/Types.Scalars/NonEmptyStringType.cs#L10 it's `NonEmptyStringType` but in the `WellKnowScalarTypes` it's  `public const string NonEmptyString = nameof(NonEmptyString);`